astroquery/esasky/core.py

```diff
@@ -738,18 +736,18 @@ def _find_mission_tap_table_name(self, json, mission_name):
         raise ValueError("Input %s not available." %mission_name)
         return None
 
-    def _get_observation_json(self, cache):
-        return self._fetch_and_parse_json(self.__OBSERVATIONS_STRING, cache)
+    def _get_observation_json(self):
+        return self._fetch_and_parse_json(self.__OBSERVATIONS_STRING)
 
-    def _get_catalogs_json(self, cache):
-        return self._fetch_and_parse_json(self.__CATALOGS_STRING, cache)
+    def _get_catalogs_json(self):
+        return self._fetch_and_parse_json(self.__CATALOGS_STRING)
 
-    def _fetch_and_parse_json(self, object_name, cache):
+    def _fetch_and_parse_json(self, object_name):
         url = self.URLbase + "/" + object_name
-        response = self._request('GET', url, cache = cache)
+        response = self._request('GET', url, cache = False)
         string_response = response.content.decode('utf-8')
         json_response = json.loads(string_response)
-        return json_response[object_name]
+        return json_response["descriptors"]
```

Comment:
It seems this is still causing  KeyError: https://travis-ci.org/astropy/astroquery/jobs/183242850#L1297